### PR TITLE
Feature log in sign up

### PR DESCRIPTION
**Pull request by nuelscode**

### CODE REVIEW

File: src/components/auth/LogIn.js [hunk 1]
```diff
@@ -3,107 +3,89 @@ import Form from 'react-bootstrap/Form';
...
-    }
+        onChangeEmail(e) {
+            this.setState({ email: e.target.value })
+        }
+    
+        onChangePassword(e) {
+            this.setState({ password: e.target.value })
+        }
+    
+
```
Comment (EngrEric) [inline src/components/auth/LogIn.js:32]: remove space

File: src/components/auth/LogIn.js [hunk 2]
```diff
@@ -3,107 +3,89 @@ import Form from 'react-bootstrap/Form';
...
+                this.setState({
+                    email: '',
+                    password: ''
+                })
+            }
+        }
+    
+        componentWillUpdate(nextProps, nextState) {
+            localStorage.setItem('user', JSON.stringify(nextState));
+        }
```
Comment (EngrEric) [inline src/components/auth/LogIn.js:52]: wrong, put this in  the onsubmit func

File: src/components/auth/SignUp.js [hunk 3]
```diff
@@ -3,45 +3,120 @@ import Form from 'react-bootstrap/Form';
...
+                    lastName: '',
+                    email: '',
+                    password: ''
+                })
+            }
+        }
+    
+        componentWillUpdate(nextProps, nextState) {
+            localStorage.setItem('user', JSON.stringify(nextState));
+        }
```
Comment (EngrEric) [inline src/components/auth/SignUp.js:68]: wrong, push to onsubmit

**Comment by EngrEric** [comment]
Nice work, why using local storage

**Comment by nuelscode** [comment]
I don't know how to set up json server yet

**Pull request merged**